File: js/remainder.js
// https://www.codewars.com/kata/524f5125ad9c12894e00003f/train/javascript

const remainder = (a,b) => {
  const smaller = Math.min(a,b);
  const larger = Math.max(a,b);

  if (smaller === 0) return NaN;

  return larger % smaller;
}

const remainderOneLine = (a,b) => a > b ? a % b : b % a;

File: python/remainder.py
# https://www.codewars.com/kata/524f5125ad9c12894e00003f/train/python

def remainder(a: int, b: int):
    smaller = min(a, b)
    larger = max(a, b)
    
    if smaller == 0:
        return None  # Return empty value if dividing by zero
    
    return larger % smaller
